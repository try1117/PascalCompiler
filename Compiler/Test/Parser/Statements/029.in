program test;
var
  a: integer = 10;
  b: integer = 20;
begin
  a := 300;
  b := a;
end.
