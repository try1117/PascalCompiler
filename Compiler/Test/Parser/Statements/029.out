test : function()
   resultType : Nil

test declarations:
   a : Integer
   |-- 10

   b : Integer
   |-- 20

|-- Statements
|            |-- :=
|            |    |-- a
|            |    --- 300
|            --- :=
|                 |-- b
|                 --- a
